buildkite/incompatible_flag_verbose_failures.py

```diff
@@ -92,11 +92,12 @@ def get_failing_jobs(build_info):
             # Recover the platform name from job command
             platform = None
             for s in command.split(" "):
-                if s.startswith("--platform="):
-                    platform = s[len("--platform="):]
+                # TODO(fweikert): Fix this once we use arbitrary task names.
```

Comment:
Looks like it's time to fix this? ;)

https://buildkite.com/bazel/bazel-at-release-plus-incompatible-flags/builds/102#b3c5b528-e4a4-4b9b-9e19-ecd35b03af21

Reply:
Seems like it :D